infrastructure/ansible/roles/dataset_loader/tasks/dataset_loader.yml

```diff
@@ -949,8 +949,8 @@
 
 - name: Snapshot all CDNs
   uri:
-    url: "{{ dl_to_url }}/api/{{ dl_to_api_version }}/cdns/{{ item.name }}/snapshot"
-    method: GET
+    url: "{{ dl_to_url }}/api/{{ dl_to_api_version }}/snapshot?cdn={{ item.name }}"
+    method: PUT
```

Comment:
Is this specific to API 3.0 or does this also work with 1.x?

Reply:
also works with 1.x -- https://traffic-control-cdn.readthedocs.io/en/latest/api/v1/snapshot_name.html